### DISCUSSION TITLE
Downtime for sx.fukt.lol

### DISCUSSION THREAD

**Discussion by iotalol** [searxng/searx-instances#271] [Report instance downtime]
'sx.fukt.lol' is currently down, don't plan on bringing it back up soon due to me being on vacation and server kernel panicking

**Comment by unixfox** [comment]
Instance removed in https://github.com/searxng/searx-instances/commit/55f4430607da769ea8c882c0f1a252076b2bdbb5.

Please notify us when it is up again so that we can add it back.

**Reply by iotalol** [reply]
Hi @unixfox I completely forgot about this thread as i've been a little busy but the instance is back up and should be for a long time.
Sorry for the inconvenience
